core/templates/dev/head/pages/learner_dashboard/LearnerDashboard.js

```diff
@@ -410,9 +411,9 @@ oppia.controller('LearnerDashboard', [
           }
         },
         controller: [
-          '$scope', '$modalInstance', '$http', 'sectionNameI18nId',
+          '$scope', '$uibModalInstance', '$http', 'sectionNameI18nId',
           'subsectionName', function(
```

Comment:
Please move `function(` to the next line.

Reply:
Done